### PR TITLE
[FIX] Unify http log

### PR DESCRIPTION
**Pull request by Aitorbp**
## Related Issues
App: https://github.com/owncloud/android/issues/4258

- [x] Added changelog files for the fixed issues in folder changelog/unreleased. More info [here](https://github.com/owncloud/android/tree/master/changelog#create-changelog-items)
_____

### CODE REVIEW

File: changelog/unreleased/4266 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Enhancement: New log format and a fix in the type handling of the type-converter
+
+http logs have been formatted to be in line with other platforms.
+On the other hand, the types of type-converter application/jrd+json have been fixed to be logable.
```
Comment (JuancaG05) [inline changelog/unreleased/4266:4]:
```suggestion
HTTP logs have been formatted to be in line with other clients. Also, the content-type `application/jrd+json` has been added to the loggable types list.
```

File: owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/http/logging/LogBuilder.kt [hunk 2]
```diff
@@ -38,5 +39,6 @@ fun MediaType?.isLoggable(): Boolean =
         (mediaType.type == "text" ||
                 mediaTypeString.contains(CONTENT_TYPE_XML) ||
                 mediaTypeString.contains(CONTENT_TYPE_JSON) ||
-                mediaTypeString.contains(CONTENT_TYPE_WWW_FORM))
+                mediaTypeString.contains(CONTENT_TYPE_WWW_FORM) ||
+                mediaTypeString.contains(CONTENT_TYPE_JRD_JSON))
```
Comment (JuancaG05) [inline owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/http/logging/LogBuilder.kt:43]: Update the documentation in the header of the class as well

**Review by JuancaG05 — CHANGES_REQUESTED** [review]
Two tiny changes here for you to review @Aitorbp!

File: owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/http/logging/LogBuilder.kt (hunk 2 above)
Comment (Aitorbp) [reply owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/http/logging/LogBuilder.kt:43]: Done

File: changelog/unreleased/4266 (hunk 1 above)
Comment (Aitorbp) [reply changelog/unreleased/4266]: Done

**Review by JuancaG05 — APPROVED** [review]
LGTM

**Comment by jesmrec** [comment]
Please, remove from Calens the part of  "New log format" because everything new in the logging is already included in an existing Calens entry. Keeping the support for the new format `application/jrd+json` is enough

**Comment by jesmrec** [comment]
Approved on my side

**Pull request merged**